### PR TITLE
Webpack reconfig

### PR DESCRIPTION
**Pull request by tzelic**

### CODE REVIEW

File: webpack-server.js [hunk 1]
```diff
@@ -3,12 +3,11 @@
 const webpack = require("webpack");
 const config = require("./webpack.config");
 const WebpackDevServer = require("webpack-dev-server");
-const path = require('path');
 
 module.exports = (PORT) => {
   const server = new WebpackDevServer(webpack(config), {
-    path: path.resolve(__dirname, './client/dist/'),
-    publicPath: '/dist/',
+    path: config.output.path,
```
Comment (b0rza) [inline webpack-server.js]: Are these parameters needed if they just mirror the webpack.config stuff?

File: client/index.html [hunk 2]
```diff
@@ -7,6 +7,6 @@
   <body>
     <script src="/webpack-dev-server.js"></script>
     <div id="app"></div>
-    <script src="dist/build.js"></script>
+    <script src="/dist/build.js"></script>
```
Comment (b0rza) [inline client/index.html]: Any specific reason for a leading slash? Seems to work fine either way?

File: webpack-server.js (hunk 1 above)
Comment (tzelic) [inline webpack-server.js]: Actually they are. On both places. Doesn't work otherwise.

File: client/index.html (hunk 2 above)
Comment (tzelic) [inline client/index.html]: Saw it on most places in docs and elsewhere so followed it like a convention while rewriting webpack configs.

**Pull request merged**